cla_backend/apps/cla_provider/tests/api/test_case_api.py

```diff
@@ -255,24 +255,46 @@ def test_closed_cases(self):
 class UpdateCaseTestCase(BaseUpdateCaseTestCase, BaseCaseTestCase):
     def test_patch_provider_notes_allowed(self):
         """
-        Test that provider can post provider notes
+        Test that provider can post provider notes up to max character limit
         """
         self.assertEqual(CaseNotesHistory.objects.all().count(), 0)
 
+        max_character_limit = "A" * CaseSerializer().fields["provider_notes"].max_length
+
         response = self.client.patch(
             self.detail_url,
-            data={"provider_notes": "abc123"},
+            data={"provider_notes": max_character_limit},
             format="json",
             HTTP_AUTHORIZATION=self.get_http_authorization(),
         )
         self.assertEqual(response.status_code, status.HTTP_200_OK)
-        self.assertEqual(response.data["provider_notes"], "abc123")
+        self.assertEqual(response.data["provider_notes"], max_character_limit)
 
         self.assertEqual(CaseNotesHistory.objects.all().count(), 1)
         case_history = CaseNotesHistory.objects.last()
-        self.assertEqual(case_history.provider_notes, "abc123")
+        self.assertEqual(case_history.provider_notes, max_character_limit)
         self.assertEqual(case_history.created_by, self.user)
 
+    def test_patch_provider_notes_not_allowed_max_limit_hit(self):
+        """
+        Test that a bad request is made when user is over max character limit.
+        """
+        self.assertEqual(CaseNotesHistory.objects.all().count(), 0)
+
+        print(CaseSerializer().fields["provider_notes"].max_length, "TEST!!")
```

Comment:
You may want to remove this print statement 😄

Reply:
OHHH!! DARN!